### ISSUE TITLE
Elementor Import tool fails when checking for permissions in a Bedrock Wordpress installation

### ISSUE THREAD

**Issue by juhojama** [elementor/elementor#20661]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The Elementor import tool fails after uploading a exported .zip file to the import tool in a Bedrock WP environment (https://github.com/roots/bedrock).

It is caused by elementor/app/modules/import-export/module.php:461 checking if the user has permission to install plugins (current_user_can( 'install_plugins' )) and throws error static::PERMISSIONS_ERROR_KEY (plugin-installation-permissions-error) when it fails.

Not being able to install plugins via WP UI is one of main features of Bedrock installation (uses Composer instead). Removing the check allows the tool to continue successfully, although it fails to detect that Elementor is already installed and tries to install it and fails.

Error log:
```
PHP: 2022-12-08 13:16:33 [error X 1][/var/www/vhosts/domain.fi/staging.domain.fi/web/app/mu-plugins/elementor/app/modules/import-export/module.php::461] Uncaught Error: plugin-installation-permissions-error in /var/www/vhosts/domain.fi/staging.domain.fi/web/app/mu-plugins/elementor/app/modules/import-export/module.php:461
Stack trace:
#0 /var/www/vhosts/domain.fi/staging.domain.fi/web/app/mu-plugins/elementor/app/modules/import-export/module.php(402): Elementor\App\Modules\ImportExport\Module->handle_upload_kit()
#1 /var/www/vhosts/domain.fi/staging.domain.fi/web/app/mu-plugins/elementor/app/modules/import-export/module.php(336): Elementor\App\Modules\ImportExport\Module->maybe_handle_ajax()
#2 /var/www/vhosts/domain.fi/staging.domain.fi/web/wp/wp-includes/class-wp-hook.php(308): Elementor\App\Modules\ImportExport\Module->Elementor\App\Modules\ImportExport\{closure}()
#3 /var/www/vhosts/domain.fi/staging.domain.fi/web/wp/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters()
#4 `/var/www/vhosts/domain.fi/staging.domain.fi/web/wp/wp-includes/plugin.php(517):` WP_Hook->do_action()
#5 /var/www/vhosts/domain.fi/staging.domain.fi/web/wp/wp-admin/admin-ajax.php(45): do_action()
#6 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
```

### Steps to reproduce

1) Export a compatible .zip file via Elementor export tool
2) Use the Elementor import tool to import the .zip in a Bedrock environment (https://github.com/roots/bedrock)

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

<html><body>

Operating System: | Linux |  
-- | -- | --
Software: | Apache |  
MySQL version: | Ubuntu 20.04 v10.3.37 |  
PHP Version: | 8.0.26 |  
PHP Memory Limit: | 128M |  
PHP Max Input Vars: | 1000 |  
PHP Max Post Size: | 8M |  
GD Installed: | Yes |  
ZIP Installed: | Yes |  
Write Permissions: | All right |  
Elementor Library: | Connected

</body>
</html>

**Comment by nicholaszein** [comment]
Hey @juhojama, 

Thank you for submitting your report! 🙏 

### ❌ While reviewing your report, we noticed it was missing some key pieces of information:

- **Isolating the Problem:** You need to make sure this only happens while using Elementor, Elementor Pro, and the Hello Elementor theme. We do not provide support for 3rd-party themes and plugins.
- **System info:** You did not provide any information about your [System Info](https://elementor.com/help/systems-info/) as outlined in our [guidelines](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml).

Without them, we simply can't help. 😕 

👉 Please follow the guidelines for opening a new [🐞 Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml), so we can help you better.

Best regards,

Nick

**Comment by juhojama** [comment]
Tested with a blank environment. This is not a 3rd party issue.

After uploading a compatible .zip:
![image](https://user-images.githubusercontent.com/24766281/206482299-3649d23a-b405-4040-a44a-572747d71508.png)

**Comment by juhojama** [comment]
Updated system information to the 1st post @nicholaszein

**Comment by nicholaszein** [comment]
Thanks for the update @juhojama! 🙏 

Can you confirm Elementor Pro is activated? Importing a Website Kit requires an active Elementor subscription. Is your license connected?

**Comment by juhojama** [comment]
Thanks for the quick follow-up @nicholaszein 

The Elementor Pro license is activated and checking the license says Status: Active

**Comment by nicholaszein** [comment]
Hello @juhojama,

I forgot to ask you, please update Elementor to v3.9.0, and Elementor Pro to v3.9.0 as well. Before we continue, we want to ensure this is and error with the current version. I see you have v3.8.1 and v3.7.7 respectively.

**Comment by nicholaszein** [comment]
Please, also check that you have added permissions correctly:

https://stackoverflow.com/questions/33095021/wordpress-you-do-not-have-sufficient-permissions-to-access-this-page-in-prod

And check if you have set the correct constants for the different directories that you have setup for plugins, wp-content, etc.

**Comment by nicholaszein** [comment]
Hello @juhojama,

any news?

**Comment by nicholaszein** [comment]
Hello @juhojama, 

#### 🕙 It's been a while since we answered you, and we haven't received any feedback. 

## 🚧 For this reason, we are closing this issue because of no activity.

👉 If you wish to continue this discussion, please follow the guidelines for opening a new [🐞 Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml).

Feel free to submit a new issue in accordance with these guidelines. 😉 

Kind regards
